talend-component-maven-plugin/src/it/web/.jenkins/scripts/server-registry-init.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
+#
+#  Copyright (C) 2006-2022 Talend Inc. - www.talend.com
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# set -xe
+
+# Configure the environment to start a local server
+# Parameters:
+[ -z ${1+x} ] && usage "Parameter 'download_dir'"
+[ -z ${2+x} ] && usage "Parameter 'install_dir'"
+[ -z ${3+x} ] && usage "Parameter 'coverage_dir'"
+[ -z ${4+x} ] && usage "Parameter 'tck_version'"
+[ -z ${5+x} ] && usage "Parameter 'connectors_version'"
+[ -z ${6+x} ] && usage "Parameter 'connector'"
+[ -z ${7+x} ] && printf "Parameter 'server_port' use the default value: 8080"
+
+DOWNLOAD_DIR=${1}
+INSTALL_DIR=${2}
+COVERAGE_DIR=${3}
+TCK_VERSION=${4}
+CONNECTOR_VERSION="${5}"
+CONNECTOR_LIST="${6}"
+SERVER_PORT=${7:-"8080"}
```

Comment:
Please use lower case variables inside the script to use local variables instead of environment variables. Unless it is required ?

Reply:
Constants are now _UPER
Var are now lower
Environment are UPER